languages/python/django-oso/tests/test_partial_parity.py

```diff
@@ -0,0 +1,165 @@
+"""Partial parity tests based on
+
+https://www.notion.so/osohq/Supported-Query-Types-and-Features-435d7a998dc14db3a125c6e5ba5fe6ba.
```

Comment:
Don't love linking to private docs in public code

Reply:
Yeah, I hear that... Not sure of an easy workaround though unless I move this to a wiki page or something on GitHub.  I think it's valuable to have that reference when going through tests.